packages/cloud/src/schema/team.ts

```diff
@@ -0,0 +1,43 @@
+import { Column, Entity, ManyToOne, OneToMany, PrimaryGeneratedColumn } from 'typeorm';
+
+import { CombatUnitSpec } from '../../../parser/dist/index';
+import { CombatStatRecord } from './combat';
+import { PlayerStatRecord } from './player';
+
+@Entity()
+export class TeamStatRecord {
+  @PrimaryGeneratedColumn()
+  rowId: number;
+
+  @Column('varchar', { length: 32 })
+  specs: string;
+
+  @Column('tinyint')
+  teamId: number;
+
+  @Column('float')
+  burstDps: number;
+
+  @Column('float')
+  effectiveDps: number;
+
+  @Column('float')
+  effectiveHps: number;
+
+  @Column({
+    type: 'enum',
+    enum: CombatUnitSpec,
+    default: CombatUnitSpec.None,
+  })
+  killTargetSpec: CombatUnitSpec;
+
+  @ManyToOne(() => CombatStatRecord, (combat) => combat.teamRecords, {
+    createForeignKeyConstraints: false,
+  })
+  combatRecord: CombatStatRecord;
```

Comment:
We could also consider flattening some of the combat record onto the team/player entities

ie: storing bracket for instance on team/player types to make the query easy; I don't think it's really a huge issue since it seems unlikely that our query times will really matter from this datasource

Reply:
Right that's exactly the tradeoff I was thinking - okay to sacrifice query time but we need to optimize for storage because the 5GB size limit is the only one that'll likely be exceeded and would make us pay. The current approach minimizes redundant storage.